src/main/java/com/google/devtools/build/lib/rules/objc/ObjcRuleClasses.java

```diff
@@ -769,8 +769,14 @@ Enables clang module support (via -fmodules).
           Setting this to 1 will allow you to @import system headers and other targets:
           @import UIKit;
           @import path_to_package_target;
+          @import module_name;
           <!-- #END_BLAZE_RULE.ATTRIBUTE -->*/
           .add(attr("enable_modules", BOOLEAN))
+          /* <!-- #BLAZE_RULE($objc_compiling_rule).ATTRIBUTE(module_name) -->
+          Controls clang module name. In swift it will apply -module-name.
```

Comment:
This does not apply to Swift anymore as the support has been removed from the native rules.